client/my-sites/patterns/components/readymade-template-details/client.tsx

```diff
@@ -0,0 +1,33 @@
+import { BlockRendererProvider, ReadymadeTemplateRenderer } from '@automattic/block-renderer';
+import { RENDERER_SITE_ID } from 'calypso/my-sites/patterns/constants';
+import { useRenderReadymadeTemplate } from 'calypso/my-sites/patterns/hooks/use-render-readymade-template';
+import { ReadymadeTemplate } from 'calypso/my-sites/patterns/types';
+import { ReadymadeTemplateDetails } from './index';
+
+const ReadymadeTemplatePreview = ( {
+	readymadeTemplate,
+}: {
+	readymadeTemplate: ReadymadeTemplate;
+} ) => {
+	const { data: renderedReadymadeTemplate = {} } = useRenderReadymadeTemplate( readymadeTemplate );
```

Comment:
Can we move `useRenderReadymadeTemplate` inside `ReadymadeTemplateRenderer`?

The `ReadymadeTemplatePreview` components looks like an unnecessary wrapper.

Reply:
I'm getting an ESLint error:
```
ESLint: 'calypso/my-sites/patterns/types' import is restricted from being used by a pattern.(no-restricted-imports)

```

Reply:
Let's leave it as it is for now. :)

Reply:
I guess `ReadymadeTemplateRenderer` shouldn't be part of the `@automattic/block-renderer` package.